third_party/data/open-cookie-database.json

```diff
@@ -4863,26 +4863,26 @@
   ],
   "lidid": [
     {
-      "platform": "liadm.com",
+      "platform": "LiveIntent",
```

Comment:
Why these have been updated?

Reply:
The CLI process (`cookie-db:update`) of updating the open cookie db JSON file made these changes.